test/lib/resources.js

```diff
@@ -100,35 +100,52 @@ for (const adapter of adapters) {
 		});
 
 		const resource = createResource({
-			path: "/dest/writer/test.js",
+			path: "/dest/writer/path/test.js",
 			string: "MyInitialContent"
 		});
 
 		await dest.write(resource);
 
-		resource.setPath("/dest/writer/test2.js");
+		resource.setPath("/dest/writer/path/test2.js");
 
-		const resourceOldPath = await dest.byPath("/dest/writer/test.js");
-		const resourceNewPath = await dest.byPath("/dest/writer/test2.js");
+		const resourceOldPath = await dest.byPath("/dest/writer/path/test.js");
```

Comment:
Now, we would have two tests testing the same functionality. @RandomByte would like to have both the intergration test "resources.js" and the unit test for the abstractadapter?

Reply:
I'm fine with that 👍